SnackOverflow/src/main/resources/db/migration/V1__Initial_schema.sql

```diff
@@ -0,0 +1,104 @@
+CREATE TABLE IF NOT EXISTS`snack_overflow`.`user` (
```

Comment:
You already have the database schema name inside the JDBC url (in the `application.properties` file, the `spring.datasource.url` property). You don't need to specify it in this file anymore. If you do specify it, then you would need to adjust this migration file if you change the schema name (which may happen e.g. between your local development system and the productive system.